numeric.c

```diff
@@ -3418,10 +3418,94 @@ rb_num2ull(VALUE val)
 
 /********************************************************************
  *
- * Document-class: Integer
+ *  Document-class: Integer
  *
- *  Holds Integer values.  You cannot add a singleton method to an
- *  Integer object, any attempt to do so will raise a TypeError.
+ *  An \Integer object represents an integer value.
+ *
+ *  An attempt to add a singleton method to this class
```

Comment:
`to this class` -> `to an instance of this class`

Reply:
Done.